dist/formats/email_alert_signup/frontend/schema.json

```diff
@@ -68,12 +66,22 @@
             "policies"
           ]
         },
+        "signup_tags": {
+          "type": "object",
+          "description": "The content item for which this is handling email signups",
+          "additionalProperties": false,
+          "properties": {
+            "policies": {
+              "type": "array"
+            },
+            "topics": {
+              "type": "array"
+            }
+          }
+        },
         "summary": {
           "type": "string"
         },
-        "tags": {
```

Comment:
Should we continue to allow "tags" for now, until we've migrated all items to use "signup_tags"?